subiquity/server/server.py

```diff
@@ -258,6 +259,7 @@ class SubiquityServer(Application):
         "Locale",
         "Refresh",
         "Kernel",
+        "KernelCrashDumps",
```

Comment:
When I see this, I sort of regret not thinking about making `crash-dump` a subkey of `kernel`. But it's a design discussion and probably too late to consider anyway. :)

Reply:
I had this thought too, but from the controller (and curtin hook) perspective it's pretty separate from the things `kernel` does. At least it made the implementation easy :grinning:

Reply:
That's fair!